utils.js

```diff
@@ -0,0 +1,14 @@
+function replaceAll(str, find, replace) {
+    return str.replace(new RegExp(find, 'g'), replace);
+}
+
+function getURLParameterByName(name) {

```

Comment:
It replaces all occurrences of "find" with "replace" in the string "str".
The 'g' means: perform a global match (find all matches rather than
stopping after the first match). Giving "replace" to the the variable name
was not a good because replace is also the method name being called. Var
name modified to "change".


2016-03-02 17:25 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In utils.js
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54785145>
> :
>
> > @@ -0,0 +1,14 @@
> > +function replaceAll(str, find, replace) {
> > +    return str.replace(new RegExp(find, 'g'), replace);
> > +}
> > +
> > +function getURLParameterByName(name) {
>
> Can you explain how this code works?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54785145>
> .
>